AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -525,7 +525,7 @@ borderlands.com##div[ads-blocked="adsBlocked"]
 ||buhgalter.com.ua/assets/templates/base/js/blocker.js
 coinhub.pw#?#body > center:contains(Turn off your adblocker)
 milehighreport.com##div[class^="adblock-whitelist-messaging"]
-sgvtribune.com,ocregister.com,bostonherald.com,adn.com#%#//scriptlet("abort-on-property-read", "MG2Loader.init")
+mercurynews.com,marinij.com,sgvtribune.com,ocregister.com,bostonherald.com,adn.com#%#//scriptlet("abort-on-property-read", "MG2Loader.init")
```

Comment:
This is good but what about content blockers?

Reply:
I can't reproduce it on iOS and CB + Samsung browser